src/components/Title.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Title = () => {
+  return <div class="font-russo text-white text-[110px]">ABOUT US</div>;
```

Comment:
change `text-[110px]`
It's a better idea to use the built in tailwind classes
https://tailwindcss.com/docs/font-size